### DISCUSSION TITLE
550.135 driver not in the repository

### DISCUSSION THREAD

**Discussion by Kiskae** [NVIDIA/open-gpu-kernel-modules#745] [Q&A] [answered]
It appears 550.135 was released 4 days ago, which fixes the build for linux 6.12: https://www.nvidia.com/en-us/drivers/details/236036/

But while it is in the nvidia CDN (https://download.nvidia.com/XFree86/NVIDIA-kernel-module-source/NVIDIA-kernel-module-source-550.135.tar.xz), it is not yet merged into this repository.

**Comment by aritger** [comment, marked as answer]
Thank you for pointing this out.  I've pushed 550.135 to the repo, now.  I apologize for the oversight.
